Example/Backpack/View Controllers/HorizontalNavSelectorViewController.swift

```diff
@@ -29,9 +29,11 @@ enum HorizontalNavSegueIdentifier: String {
     case showExtraContent = "withScroll"
     case customItems = "customItems"
     case notificationDot = "withNotification"
+    case badge = "withBadge"
 }
 
 class HorizontalNavSelectorViewController: UITableViewController {
+    // swiftlint:disable:next function_body_length cyclomatic_complexity
```

Comment:
Maybe it's time to refactor this instead of disabling the linting rule?

Reply:
Can we do that on a separate PR as EngHealth?

Reply:
Sounds good yeah